adagrams/game.py

```diff
@@ -1,11 +1,113 @@
+import random
+
+LETTER_POOL = {
+    'A': 9,
+    'B': 2,
+    'C': 2,
+    'D': 4,
+    'E': 12,
+    'F': 2,
+    'G': 3,
+    'H': 2,
+    'I': 9,
+    'J': 1,
+    'K': 1,
+    'L': 4,
+    'M': 2,
+    'N': 6,
+    'O': 8,
+    'P': 2,
+    'Q': 1,
+    'R': 6,
+    'S': 4,
+    'T': 6,
+    'U': 4,
+    'V': 2,
+    'W': 2,
+    'X': 1,
+    'Y': 2,
+    'Z': 1
+}
+
+SCORE_CHART = {
+    1: ['a', 'e', 'i', 'o', 'u', 'l', 'n', 'r', 's', 't'],
+    2: ['d', 'g'],
+    3: ['b', 'c', 'm', 'p'],
+    4: ['f', 'h', 'v', 'w', 'y'],
+    5: ['k'],
+    8: ['j', 'x'],
+    10: ['q', 'z'],
+}
+
+
 def draw_letters():
-    pass
+    letter_hand = []
+    letter_pool_list = []
+    for letter, number in LETTER_POOL.items():
+        letter_pool_list.extend([letter] * number)
```

Comment:
Great use of `extend` to create a list of letters according to their letter count in the LETTER_POOL. You can omit the brackets around `letter` as well and it will create that same single-dimension list. Was the intention to create a nested list? 

```suggestion
        letter_pool_list.extend(letter * number)
```